### DISCUSSION TITLE
Log syslog to Docker logs command

### DISCUSSION THREAD

**Discussion by akama-aka** [docker-library/postgres#1264] [General]
Hello,
I have a problem right now and my PostgreSQL container is in a restarting loop due to an error. Via docker logs <container id> -f I only get outdated logs and nothing more and otherwise it is not possible for me to get any more logs. 

In my opinion it is a fatal problem within the image as it doesn't allow you to perform an update of the file as it is not possible to connect to the container while the maintainer is in a restart state.

Translated with DeepL.com (free version)

**Comment by tianon** [comment]
Unfortunately, logging of your containers is something you'll need to set up on your host, not something we can really meaningfully affect from the image (nor could we choose a "sane" default that would satisfy our diverse userbase).
